app/templates/pdf/order_invoice.html

```diff
@@ -218,19 +223,37 @@ <h1>{{ _('INVOICE') }}</h1>
           <tr>
             <td class="service">{{ order_ticket.ticket.name }}</td>
             <td class="total">
-                {{ event.payment_currency }}{{ order_ticket.ticket.price }}
+              {% if order_ticket.ticket.price %}
+                {{ event.payment_currency }}&nbsp;
+              {% endif %}
+              {{ "%.2f" | format(order_ticket.ticket.price) }}
             </td>
             <td>{{ order_ticket.quantity }}</td>
-            <td class="total">{{ event.payment_currency }}{{ order_ticket.quantity*order_ticket.ticket.price }}</td>
+            <td class="total">
+              {% if order_ticket.ticket.price %}
+                {{ event.payment_currency }}&nbsp;
+              {% endif %}
+              {{ "%.2f" | format(order_ticket.quantity*order_ticket.ticket.price) }}
+          </td>
             {% if tax %}
                 <td>{{ tax.rate }}%</td>
             {% else %}
                 <td>{{ ("0%") }}</td>
             {% endif %}
             {% if tax %}
-                <td class="total">{{ event.payment_currency }}{{ tax.rate*order_ticket.ticket.price/100 }}</td>
+                <td class="total">
+                  {% if order_ticket.ticket.price %}
+                    {{ event.payment_currency }}&nbsp;
+                  {% endif %}
+                  {{ tax.rate*order_ticket.ticket.price/100 }}
+                </td>
             {% else %}
-                <td class="total">{{ event.payment_currency }}0</td>
+                <td class="total">
+                  {% if order_ticket.ticket.price %}
+                    {{ event.payment_currency }}&nbsp;
+                  {% endif %}
```

Comment:
why add this? if below-mentioned amount is already 0.00

Reply:
right